src/zabapgit_authorizations.prog.abap

```diff
@@ -0,0 +1,49 @@
+*&---------------------------------------------------------------------*
+*&  Include           ZABAPGIT_AUTHORIZATIONS
+*&---------------------------------------------------------------------*
+
+TYPES: ty_activity TYPE string.
+
+CONSTANTS: BEGIN OF gc_activities,
+             uninstall TYPE ty_activity VALUE 'UNINSTALL',
+           END OF gc_activities.
```

Comment:
perhaps `gc_authorization`? I choose activity because it is often used in connection with SAP standard authorization objects